.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
Ran `npm install` and made an edit in StartScreen to import an unused var: 

<img width="792" alt="image" src="https://user-images.githubusercontent.com/63211322/205460401-0692da34-7e29-48c9-bbce-49c1772a18a0.png">

Committing this change yields:
<img width="591" alt="image" src="https://user-images.githubusercontent.com/63211322/205460449-c2c13fc1-23ef-4720-a543-4a4f804210a5.png">

Are there other steps I need to take to setup the pre-commit hooks?